sql/augments.sql

```diff
@@ -1427,8 +1427,8 @@ INSERT INTO `augments` VALUES (1151,0,0,0,0,0);
 INSERT INTO `augments` VALUES (1152,0,1,10,0,0); -- DEF +10
 INSERT INTO `augments` VALUES (1153,0,68,3,0,0); -- Evasion +3
 INSERT INTO `augments` VALUES (1154,0,31,3,0,0); -- Mag. Evasion +3
-INSERT INTO `augments` VALUES (1155,100,161,-2,0,0); -- Physical Damage Taken -2%
-INSERT INTO `augments` VALUES (1156,100,163,-2,0,0); -- Magic Damage Taken -2%
+INSERT INTO `augments` VALUES (1155,200,161,-1,0,0); -- Physical Damage Taken -2%
+INSERT INTO `augments` VALUES (1156,200,163,-1,0,0); -- Magic Damage Taken -2%
```

Comment:
can you take a look at aug ID 71 also? thanks.

Reply:
![image](https://github.com/LandSandBoat/server/assets/131182600/30f8171a-f7a2-4a3a-93a2-bf5e739dd6c4)

🔨

Reply:
found those 3 more augments and fixed to match the rest of what this PR is doing. Essentially some augments do have a base value then increment by 1, but _every_ mod that we (lsb) chooses to have some other base on the values has to use mutlipliers even if they increment "by one" as in these extra 3.

TL;DR - the 3 above had this behavior:
![image](https://github.com/LandSandBoat/server/assets/131182600/6c121bdf-2200-47a6-a31a-9bd91652d59e)

Reply:
thanks.